modules/contact-form/css/gutenblock.css

```diff
@@ -0,0 +1,84 @@
+
+
+.grunion-field-label.grunion-field-label.grunion-field-label {
+	border-color: #fff;
+	font-weight: bold;
+	margin: 0;
+	margin-bottom: 2px;
+	padding-left: 1px;
+	width: calc( 100% - 100px );
+}
+
+.grunion-field-label.grunion-field-label.grunion-field-label:focus,
+.grunion-field-label.grunion-field-label.grunion-field-label:hover {
+	border-color: #8d96a0;
+}
+
+.grunion-field input:disabled,
+.grunion-field textarea:disabled {
+	border-color: #8d96a0;
+	padding: 10px 8px;
+}
+
+.grunion-form .components-base-control * {
+	margin: 0;
+}
+
+span.required {
```

Comment:
This one looks like it needs `grunion-` prefix as it would end up being global and affect things outside the block.

Reply:
That's fair.